Servers/UI/OJS.Servers.Ui/ClientApp/src/styles/components.scss

```diff
@@ -32,4 +32,35 @@
   @media only screen and (max-width: 1440px) {
     width: 1280px;
   }
-}
\ No newline at end of file
+}
+
+.sideNavigation {
+  align-items: start;
+  padding: 20px;
+
+  .sideNavigationItem {
+    text-align: start;
+    font-size: 18px;
+    font-family: 'Lato', serif;
+    color: $text-dark-gray-color;
```

Comment:
Is this the correct place for these? As I see above, it has only generic stuff, i.e. `small`, `large`, `contentSize`, etc...

`sideNavigation` seems off here.